### ISSUE TITLE
Question about .ToObject<T>() method.

### ISSUE THREAD

**Issue by rdcm** [cemdervis/SharpConfig#51]
Hi!

I have this config file:

```
[Person]
Name = Foo
Age = 234
Gender = Male
```

In the code there are entities:
```
public class Person
{
	public string Name { get; set; }
	public int Age { get; set; }
	public Gender Gender { get; set; }
}

public enum Gender
{
	Male,
	Female
}
```
Main method:
```
var config = Configuration.LoadFromFile(@"....config.cfg");
var obj = config["Person"].ToObject<Person>();
```
`When i invoke .ToObject<Person>(); this method throw exception:`
```
SharpConfig.SettingValueCastException: 
Failed to convert value 'Male' to type cfg.Gender; no converter for this type is registered. ---> 
System.NotImplementedException: no converter for this type is registered.
   --- End of inner exception stack trace ---
   at SharpConfig.Setting.CreateObjectFromString(String value, Type dstType)
   at SharpConfig.Setting.GetValue(Type type)
   at SharpConfig.Section.SetValuesTo(Object obj)
   at SharpConfig.Section.ToObject[T]()
   at cfg.Program.Main(String[] args) in C:\....\Program.cs:line 19
```

If i remove 'Gender' field, all works perfect. Looks like a bug :)

**Comment by cemdervis** [comment]
Hey rdcm, have you tried using the latest source code of SharpConfig, or is this with version 3.0.1?
This bug was fixed with commit 95a3962. I'll release a version soon.

**Comment by rdcm** [comment]
Yes, i use latest NuGet package version 3.0.1. 
Ok, thanks, will be wait update :)

**Comment by cemdervis** [comment]
Fixed with version 3.1.0!
